### ISSUE TITLE
Allow progress info while downloading

### ISSUE THREAD

**Issue by GuillaumeLeclerc** [Stuk/jszip-utils#14]
The XHR object allow you to know the progess of the download. It might be a good thing to be able to access it using this.

**Comment by victornpb** [comment]
I implemented this on a fork https://github.com/Stuk/jszip-utils/pull/19 and submitted it as a PR.

**Comment by ufukomer** [comment]
Is not there any solution for this yet?

**Comment by Stuk** [comment]
Just a *bit* of a delay. I've reviewed the PR.

**Comment by Stuk** [comment]
Released as version v0.1.0.

Pass an options object with a `progress` callback:

```js
JSZipUtils.getBinaryContent("ref/amount.txt", {
    progress: function(e) {
        console.log(e.percent + "% loaded");
    },
    callback: function (err, data) {
        // ...
    }
});
```
